modules/guides/pages/kv-operations.adoc

```diff
@@ -0,0 +1,310 @@
+= Key Value Operations
+:description: How to perform CRUD key value operations in Couchbase.
+:page-pagination: next
+:page-topic-type: guide
+:page-toclevels: 2
+:tabs:
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+[abstract]
+{description}
+
+== Introduction
+
+Every item in a database goes through the basic _CRUD_ cycle, which is typical of an application’s use of data.
+CRUD stands for create, read, update, and delete:
+
+* **C**reate: when data is first inserted into the cluster
+* **R**ead: when an application retrieves the data
+* **U**pdate: when data is modified to reflect a change in the state represented by the data
+* **D**elete: when the data is no longer needed
+
+The xref:learn:services-and-indexes/services/data-service.adoc[Key Value(KV) or Data Service] offers Couchbase clients the fastest and simplest way to create, retrieve or mutate data where the key is known.
+
+include::partial$before-you-begin.adoc[]
+
+include::partial$clients.adoc[]
+
+The following examples demonstrate the basic CRUD operations.
+
+== Creating Documents
+
+Creating a document means inserting new data into the database.
+The example below shows you how to insert a document with ID `airport-123`.
```

Comment:
This example is great! As discussed with @maria-robobug we don't have consistent docs per SDK on how to insert JSON, and this dev-reorg tutorial is a great place to put this.

Note that the example *doesn't* cover:

- booleans
- arrays
- nulls
- any non-JSON objects (dates)

This is something we'll try to do in the SDK docs, but just mentioning in case any of those would be useful to consider for a future iteration.